lib/app/modules/home/widgets/navbar/navbar_widget.dart

```diff
@@ -34,6 +34,17 @@ class _NavbarWidgetState extends ModularState<NavbarWidget, NavbarController> {
                 ),
               ),
             ),
+            TextButton(
+              onPressed: () async {
+                await controller.urlLauncher.launchUrl(
+                  'https://forms.gle/6BoxwQLLBE2vZeMk6',
+                );
+              },
```

Comment:
There's no need to `async-await` in this case, as it's the same thing as if not used.

```suggestion
              onPressed: () {
                controller.urlLauncher.launchUrl(
                  'https://forms.gle/6BoxwQLLBE2vZeMk6',
                );
              },
```